src/components/calcite-tree-item/calcite-tree-item.e2e.ts

```diff
@@ -174,4 +174,26 @@ describe("calcite-tree-item", () => {
       ancestors.forEach(async (node) => expect(await node.getProperty("indeterminate")).toBe(true));
     });
   });
+
+  describe("CSS Custom Properties for style overrides", () => {
```

Comment:
This is great. I think we could add a test util for this (outside of this PR). Ideally a spec test since we don't need interaction.